src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -64,6 +65,11 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>Resource1.resx</DependentUpon>
     </Compile>
+    <Compile Include="Resources.Designer.cs">
```

Comment:
Yes, I noticed resource1.resx, and I was going to rename it in a future commit as soon as I figured out what it was for :-) But now I'll just leave it to you.